tests/urls.php

```diff
@@ -64,37 +69,32 @@ public function test_event_translations() {
 	}
 
 	public function test_event_edit() {
-		$event_id = $this->event_factory->create_active();
-
+		$event_id = 42;
```

Comment:
Maybe worth adding a comment here why we don't need a real event here?

Reply:
We're testing the `Urls` class which doesn't interact with the database, it takes the event id as argument, and returns the URL. It will work regardless of whether the event exists in the database or not.

I think what happened was that we copy-pasted from another test that was testing the slug, and since the slug is calculated automatically upon event creation, in those tests we do rely on creating the event to get the slug.